statsmodels/tsa/vector_ar/svar_model.py

```diff
@@ -12,6 +12,10 @@
 import numpy as np
 import numpy.linalg as npl
```

Comment:
Getting rid of this and other unique `import as [...]` names in this modules would be nice